CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ to [Semantic Versioning](http://semver.org/).
     accessible.
 
 ### Changed
+- Homepage has been refreshed to better inform
```

Comment:
Ack lol I didn't actually look to make sure your new bullet points were being added to the "Unreleased" section--you added them to the release notes for our last release :)  Fixed this in e3ac6288f5231dcfdce6245f003a29acea25edf8